templates/dhcpd.pool.erb

```diff
@@ -20,14 +20,14 @@ subnet <%= @network %> netmask <%= @mask %> {
 <%   @options.each do |opt| -%>
   option <%= opt %>;
 <%   end -%>
-<% elsif @options != '' -%>
+<% elsif @options and !@options.empty? -%>
```

Comment:
In the installer (puppet templates) and is more common right now. I'm not against it, but then we should do it for all modules.